pkg/pillar/types/assignableadapters.go

```diff
@@ -36,14 +37,17 @@ type IoBundle struct {
 	//	be assigned to an application.
 	Name string
 
+	// Logical Label assigned to the Adapter
+	Logicallabel string
+
 	// Assignment Group, is unique label that is applied across PhysicalIOs
 	// Entire group can be assigned to application or nothing at all
 	AssignmentGroup string
 
-	// UsedByUUID
-	//	Application UUID ( Can be Dom0 too ) that owns the Bundle.
-	//	For unassigned adapters, this is not set.
-	UsedByUUID uuid.UUID
+	Usage zconfig.PhyIoMemberUsage
+
+	// Used only when one other normal uplinks are down
```

Comment:
Comment is not correct. The policy is more complex than that.

Reply:
Changed it to:

	// FreeUplink - The network connection through this adapter is Free.
	// Prefer this adapter for connecting to the cloud.